### PR TITLE
Starting benchmarks

### PR DESCRIPTION
**Pull request by levkk**
Good start.

### CODE REVIEW

File: pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+### Train
+
+```postgresql
+SELECT * FROM pgml.train(
+	'r2',
+	'regression',
+	'flights_delay_mat',
+	'depdelayminutes',
+	algorithm => 'xgboost',
+	hyperparams => '{ "n_estimators": 25 }'
```
Comment (montanalow) [inline pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/README.md:46]: It'd be interesting to see how scaling the estimators impacts performance in both (10/100/1000 estimators)

File: pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/load_redis.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+	df = pd.read_csv("~/Desktop/flights_sub.csv")
+
+	print("Uploading data to feature store")
+
+	pipeline = r.pipeline()
+	i = 0
+
+	# Upload data to feature store
+	for row in df.iterrows():
```
Comment (montanalow) [inline pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/load_redis.py:17]: It'll be fun to have a "data loading" benchmark as well.

File: pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/pgbench.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+SELECT pgml.predict('r2',
+	ARRAY[year, quarter, month, distance, dayofweek, dayofmonth, flight_number_operating_airline, originairportid, destairportid, tail_number]
+) FROM flights_delay_mat WHERE originairportid = 14869 and year = 2019 and month = 8 and dayofmonth = 31 LIMIT 1;
```
Comment (montanalow) [inline pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/pgbench.sql:3]: Similarly, it'd be nice to have batch sizes of 1/10/100/1000 here as well, to see scaling.

File: pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/load_redis.py (hunk 2 above)
Comment (levkk) [reply pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/load_redis.py:17]: For sure, it's super slow here.

File: pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/pgbench.sql (hunk 3 above)
Comment (levkk) [reply pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/pgbench.sql:3]: We don't have a batch API in PG at the moment, but if you want to play unfair, set the limit higher. I think we have about 250 rows for that airport and date.

File: pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/README.md (hunk 1 above)
Comment (levkk) [reply pgml-docs/docs/blog/benchmarks/python_microservices_vs_postgresml/README.md:48]: More estimators, more problems.

**Pull request merged**